packages/system/src/components/homepage/contributor-banner.css.ts

```diff
@@ -13,21 +12,24 @@ export const contributorBannerStyle = style([
 			tablet: "grid",
 		},
 		flexDirection: "column",
-		alignItems: "center",
+		alignItems: "start",
 	}),
 	{
+		backgroundColor: "transparent",
 		gridTemplateColumns: "1fr 3fr min-content",
 	},
 ])
 
-export const contributorImageStyle = style([
-	sprinkles({ borderColor: "onSecondaryContainer" }),
+const contributorsSpacingPx = 4
+export const contributorsContainerStyle = style([
 	{
-		maxWidth: pxToRem(72),
-		borderRadius: "50%",
-		borderWidth: pxToRem(4),
-		borderStyle: "solid",
-		marginLeft: pxToRem(5),
+		margin: pxToRem(-contributorsSpacingPx / 2),
```

Comment:
Since we are only using this as `contributorsSpacingPx/2`. Could we instead set `contributorsSpacingPx` to 2` and then in the `pxToRem` function, have `-contributorsSpacing` and `contributorSpacing`?

Reply:
Actually, I made this intentionally to make the variable controlling the actual result that user will see. Since we are adding the same margin on all adjacent elements the spacing between them would be twice the margin. That's why by setting desired spacing to be 4px I need to set margin to 2px

Reply:
Actually, I made this intentionally to make the variable controlling the actual result that user will see. Since we are adding the same margin on all adjacent elements the spacing between them would be twice the margin. That's why by setting desired spacing to be 4px I need to set margin to 2px

Reply:
Ah, you mean semantically, since we have multiple components with `2px` margins, then the space between the elements is actually `4px`? In that case then this does make sense. Looks good!